djonet/creation.py

```diff
@@ -54,7 +54,7 @@ def monetdb_control(self):
                                        self.monetdb_port,
                                        self.monetdb_passphrase)
 
-    def _create_test_db(self, verbosity, autoclobber):
+    def _create_test_db(self, verbosity, autoclobber, keepdb=False):
```

Comment:
It would be nice to use it too. Just `if keepdb: return` in except clause at least.